### PR TITLE
Addressed feedback

### PR DESCRIPTION
**Pull request by DmitryVasilevsky**
Addressed feedback by Mariia Mykhailova and Brad Benefield.

### CODE REVIEW

File: katas/content/oracles/index.md [hunk 1]
```diff
@@ -397,8 +397,8 @@ Here we will compare the reference implementation of `Meeting_Classical` oracle
 
 Congratulations! In this kata you have learned to build quantum oracles. Here are a few key concepts to keep in mind:
 * A quantum oracle is an "opaque box" operation that is used as input to another algorithm.
-* Phase oracles encode the information in the relative phase of basis states $\ket{0}$ and $\ket{1}$. If $f(x)=0$, it does nothing, and if $f(x)=1$ it multiplies the phase of the basis states for which by $-1$.
-* Marking oracles uses an extra qubit $\ket{y}$ and encode the information in the state of that qubit. If $f(x)=0$, it does nothing, and if $f(x)=1$ it it flips the state of the qubit $\ket{y}$.
+* Phase oracles encode the information in the relative phase of basis states. If $f(x)=0$, the oracle doesn't change the basis state $\ket{x}$, and if $f(x)=1$ it multiplies the phase of the basis states $\ket{x}$ by $-1$.
```
Comment (tcNickolas) [inline katas/content/oracles/index.md:400]:
```suggestion
* Phase oracles encode the information in the relative phase of basis states. If $f(x)=0$, the oracle doesn't change the basis state $\ket{x}$, and if $f(x)=1$ it multiplies the phase of the basis state $\ket{x}$ by $-1$.
```
(we're looking at one x at a time, same as in the next bullet)

File: katas/content/random_numbers/index.md [hunk 2]
```diff
@@ -96,13 +96,14 @@ This knowledge is sufficient to implement a simple random number generator!
 
 @[section]({"id": "random_numbers__whats_next", "title": "What's Next?"})
 
-Congratulations! In this Kata you have created a truly random number generator using quantum computing. Here are a few key concepts to keep in mind:
+Congratulations! In this kata you have created a truly random number generator. Here are a few key concepts to keep in mind:
```
Comment (tcNickolas) [inline katas/content/random_numbers/index.md:99]:
```suggestion
Congratulations! In this kata you have created a random number generator. Here are a few key concepts to keep in mind:
```
(We're telling them that it's not truly random in the next line)

**Review by tcNickolas — APPROVED** [review]
Looks good, just two small things
(That reminds me to do matching changes to the original katas...)

File: katas/content/random_numbers/index.md (hunk 2 above)
Comment (DmitryVasilevsky) [reply katas/content/random_numbers/index.md]: No more pride moments :(

**Review by billti — APPROVED** [review]

**Pull request merged**